### PR TITLE
Setting up App Analytics 🫀 

### PR DESCRIPTION
**Pull request by jumaallan**
Story: https://app.shortcut.com/smileid/story/xxx

## Summary

A few sentences/bullet points about the changes

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

### CODE REVIEW

**Review by wangerekaharun — APPROVED** [review]

File: analytics/src/main/java/com/smileidentity/analytics/AnalyticsEvent.kt [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        }
+    }
+
+    /**
+     * A key-value pair used to supply extra context to an analytics event.
+     *
+     * @param key - the parameter key. Wherever possible use one of the standard `ParamKeys`
+     * @param value - the parameter value.
+     */
+    data class Param(val key: String, val value: String)
```
[the remark this answers is not in the document]
Comment (jumaallan) [reply analytics/src/main/java/com/smileidentity/analytics/AnalyticsEvent.kt:25]: the timestamp could be a key and the value the time
